packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -25,6 +25,7 @@
   ],
   "dependencies": {
     "@salesforce/salesforcedx-apex-debugger": "57.11.1",
+    "@salesforce/salesforcedx-utils-vscode": "57.11.1",
```

Comment:
In the [isvContext](https://github.com/forcedotcom/salesforcedx-vscode/blob/9539e53525085f0f6485524d35442687a90168ff/packages/salesforcedx-vscode-apex-debugger/src/context/isvContext.ts#L8) we use utils-vscode to get the project paths.  I'm not sure why the `check:typescript-project-references` check doesn't beef about the missing dep